formatter/types.go

```diff
@@ -0,0 +1,50 @@
+package formatter
+
+type Output struct {
+	Results []Results `json:"Results"`
```

Comment:
Does it make sense to create a new structure?
IIUC we can use `types.Report` from Trivy (https://github.com/aquasecurity/trivy-plugin-count/blob/b98cf0645db896053481c4faf8bc2724c6ae39b1/main.go#L23)

Reply:
updated in https://github.com/dramf/md/pull/3/commits/81abc1dff8a5af03f8b136536ef90b49a9f6a3da